test/orchard/apropos_test.clj

```diff
@@ -2,8 +2,8 @@
   (:require
    [clojure.repl :as repl]
    [clojure.string :as str]
-   [clojure.test :refer :all]
-   [orchard.apropos :refer :all]
+   [clojure.test :refer [are deftest is testing]]
+   [orchard.apropos :refer [find-symbols]]
```

Comment:
Perhaps it'd be more idiomatic to:

```diff
-orchard.apropos :refer [find-symbols]]
+orchard.apropos :as sut]
```

Refs:

https://en.wikipedia.org/wiki/System_under_test
https://github.com/clojure-emacs/clj-refactor.el/blob/9f88174878d62e3906be5b04b8ba7788e6ca4570/clj-refactor.el#L220

Reply:
I agree this makes sense in general, but I think it's ortogonal to this PR.